js/src/launchAsync.ts

```diff
@@ -274,6 +276,22 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             src += '&omkt=' + options.uiLang;
         }
 
+        // The default value for disableLanguageDetection is false, and is used too if option is missed.
```

Comment:
update comment - , set option only when is true

Reply:
done